Description: Remove chrome_browser_browser from Debian skip list
 The chrome_browser_browser patch has been updated for Debian Chromium .96
 and should now apply cleanly.
Author: Juan Manuel Méndez Rey <vejeta@gmail.com>
Bug-Debian: https://bugs.debian.org/915400
Forwarded: not-needed
Last-Update: 2026-01-24

diff --git a/cef/tools/patcher.py b/cef/tools/patcher.py
index 6797052..9877ad7 100644
--- a/cef/tools/patcher.py
+++ b/cef/tools/patcher.py
@@ -26,7 +26,6 @@ def write_note(type, note):
   sys.stdout.write(separator)
   sys.stdout.write('!!!! %s: %s\n' % (type, note))
   sys.stdout.write(separator)
-  sys.stdout.flush()
 
 
 def apply_patch_file(patch_file, patch_dir):
@@ -61,7 +60,7 @@ def apply_patch_config():
 
   # Parse the configuration file.
   scope = {}
-  exec(compile(open(config_file, "rb").read(), config_file, 'exec'), scope)
+  exec (compile(open(config_file, "rb").read(), config_file, 'exec'), scope)
   patches = scope["patches"]
 
   results = {'apply': [], 'skip': [], 'fail': []}
@@ -88,7 +87,6 @@ def apply_patch_config():
 
   sys.stdout.write('\n%d patches total (%d applied, %d skipped, %d failed)\n' % \
       (len(patches), len(results['apply']), len(results['skip']), len(results['fail'])))
-  sys.stdout.flush()
 
   if len(results['fail']) > 0:
     sys.stdout.write('\n')
